docs/getting-started-guides/rkt/notes.md

```diff
@@ -1,7 +1,7 @@
 ---
 ---
 
-# Known issues
+# Known Issues
```

Comment:
Remove this heading altogether; the page title (as defined in the .yml file) will be a big <h1> at the top here.